Sprintfinity3902/Sprites/Animation/Animation.cs

```diff
@@ -64,7 +64,6 @@ public class Animation {
 
                 if (PlaybackProgress > Duration) {
                     if (PlayOneTime) {
-                        PlaybackProgress -= Duration;
```

Comment:
I think you might have removed this line to try to fix the ItemSelectIconSprite... but I think it should be added back in.